images/linux/scripts/installers/ansible.sh

```diff
@@ -6,11 +6,15 @@
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
-source $HELPER_SCRIPTS/apt.sh
+source $HELPER_SCRIPTS/os.sh
 
-# Install Ansible PPA and latest Ansible
-add-apt-repository ppa:ansible/ansible
-apt-get update
+# ppa:ansible/ansible doesn't contain packages for Ubuntu20.04
+if isUbuntu16 || isUbuntu18 ; then
```

Comment:
this smells, why are we installing from PPA? Aren't those unofficial?

Reply:
Ansible provides this method actually
![image](https://user-images.githubusercontent.com/48208649/82494194-e059d180-9af1-11ea-9714-53e938f9758d.png)

Reply:
ok. i guess that's how it's hosted. that's fine then. thank you!